packages/flutter_tools/test/general.shard/runner/flutter_command_runner_test.dart

```diff
@@ -17,6 +19,7 @@ import 'package:flutter_tools/src/version.dart';
 
 import '../../src/common.dart';
 import '../../src/context.dart';
+import '../../src/mocks.dart';
```

Comment:
Presubmit checks are failing because this is reported as an unused import.

Reply:
Removed